src/ziggurat/mapper.clj

```diff
@@ -49,13 +54,17 @@
                 end-time    (.toEpochMilli (Instant/now))]
             (metrics/report-time (str metric-namespace ".execution-time") (- end-time start-time))
             (case return-code
-              :success (metrics/increment-count message-processing-namespace "success")
-              :retry (do (metrics/increment-count message-processing-namespace "retry")
-                         (producer/retry-for-channel message topic-entity channel))
-              :skip (metrics/increment-count message-processing-namespace "skip")
-              :block 'TODO
+              :success (do (metrics/increment-count message-processing-namespace "success" topic-entity-name)
```

Comment:
First argument can go as vector of string and then we can concat that vector in metrics namespace.
That would be lot cleaner. I think

Reply:
done